pkg/controller/directvolumemigration/rsync.go

```diff
@@ -1684,15 +1798,17 @@ func (req rsyncClientPodRequirements) getRsyncClientPodTemplate() corev1.Pod {
 
 func (t *Task) prepareRsyncPodRequirements(srcClient compat.Client) ([]rsyncClientPodRequirements, error) {
 	req := []rsyncClientPodRequirements{}
-	cluster, err := t.Owner.GetSourceCluster(t.Client)
-	if err != nil {
-		return req, liberr.Wrap(err)
-	}
+	//cluster, err := t.Owner.GetSourceCluster(t.Client)
+	//if err != nil {
+	//	return req, liberr.Wrap(err)
+	//}
 	t.Log.Info("Getting image for Rsync client Pods that will be created on source MigCluster")
-	transferImage, err := cluster.GetRsyncTransferImage(t.Client)
-	if err != nil {
-		return req, liberr.Wrap(err)
-	}
+	//transferImage, err := cluster.GetRsyncTransferImage(t.Client)
+	//if err != nil {
+	//	return req, liberr.Wrap(err)
+	//}
+	// will update this once the rsync-trasnfer PR is merged
+	transferImage := "quay.io/spampatt/rsync-transfer:latest"
```

Comment:
drop the comments? the transferImage variable on migrationcontroller can be used for testing?